docs/integrations/custom/woocommerce.md

```diff
@@ -0,0 +1,66 @@
+# WooCommerce Integration
+
+###Setup your Metisa account
+Create your login name, team name, and team URL at https://askmetisa.com/create. You should receive an email with instructions to create a password and login.
+
+### Customise your theme templates
+You can safely customise the theme templates of your store using **overrides**. Details can be found [here](https://docs.woocommerce.com/document/template-structure/).
+
```

Comment:
I suggest adding a short paragraph here to explain that they need to SSH into their file system in order to add and edit files (perhaps WC has their own article on this, just link it). Most people probably don't know how to go about doing it, so this is crucial

Reply:
May I know the link of the article? I didn't find it in their website.